app/include/zmk/sensors.h

```diff
@@ -6,6 +6,9 @@
 
 #pragma once
 
+#include <zephyr/drivers/sensor.h>
+
+#define _SENSOR_CHILD_LEN(node) 1 +
```

Comment:
Identifiers beginning with an underscore followed by an uppercase letter are reserved for the compiler.

Reply:
Isn't actually used any more, so removed it.